config/rbac/role.yaml

```diff
@@ -54,6 +54,17 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - slack.stakater.com/v1alpha1
```

Comment:
this is defined twice in this file. Either clean up this one or the one defined at the end.

Reply:
I am testing , current code is not working ... once it does will remove the extra one

Reply:
ok, the PR is up for merge yet or no? 
if yes, please test it out with which one works and then remove the other before merging it please.

Reply:
done